### ISSUE TITLE
[Enhancement]: Automatically check for changes in files when reloading the project

### ISSUE THREAD

**Issue by zkewind** [Pythagora-io/gpt-pilot#907]
### Version

VisualStudio Code extension

### Suggestion

When reloading a projecgt, there is no need to ask if there have been any changes to the code or requirements. Simply check the file timestamp and file size to confirm if there have been any changes.

**Comment by MaxAnfilofyev** [comment]
maybe check hashes instead

**Comment by zkewind** [comment]
> maybe check hashes instead

Yes. There are many projects to refer to, such as GPT-engineer and auto-code.
